src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -637,11 +636,10 @@ private void PrepareAuthorityInstanceForMsal()
             }
             catch (MsalUiRequiredException ex)
             {
-                _logger.LogInformation(
-                    string.Format(
-                        CultureInfo.InvariantCulture,
-                        LogMessages.ErrorAcquiringTokenForDownstreamWebApi,
-                        ex.Message));
+                Logger.TokenAcquisitionError(
+                    _logger,
+                    LogMessages.ErrorAcquiringTokenForDownstreamWebApi + ex.Message,
```

Comment:
It's probably re-written with the exception, but I guess yes, it does not harm to have the Message as well in the error string

Reply:
this is what we were doing before, so tried to keep it the same, but it can be removed.